src/sdk/models/policyStatistics.ts

```diff
@@ -61,3 +61,14 @@ export interface PolicyStatistics extends IEntityIdentifier<string> {
 	by_group?: LabelledStatisticList<PolicyStatistic>;
 	by_audit_category?: LabelledStatisticList<LocationStatistic>;
 }
+
+export interface PolicySummaryByRisk {
+	bytes: number;
+	files: number;
+	name: string;
+	order: number;
+}
+
+export interface PoliciesStatisticsSummary {
+	policies_by_risk?: PolicySummaryByRisk[];
```

Comment:
I think this can be
```suggestion
	policies_by_risk?: PolicyStatistic[];
```

But you would need to add `policy?: Policy` prop [here](https://github.com/portalarchitects/skysync-cli/blob/ff24083784964a3f2640bdc934167a0207055530/src/sdk/models/policyStatistics.ts#L38)